src/pubsub.js

```diff
@@ -50,6 +62,94 @@ class BasicPubSub extends Pubsub {
     })
   }
 
+  /**
+   * Dial a received peer.
+   * @override
+   * @param {PeerInfo} peerInfo The peer being dialed
+   * @param {function} callback
+   *
+   * @returns {void}
+   */
+  _dialPeer (peerInfo, callback) {
+    const onDial = (idB58Str, pubsubStopped, conn) => {
```

Comment:
I'm not sure I understand the purpose of this function wrapper for `_onDial`

Reply:
I moved the logic to the `_onDial` function.